<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ppl_asset_win</class>
 <widget class="QMainWindow" name="ppl_asset_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>前期Asset 整理检查</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="m_tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>文件整理</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="pick_sel_l"/>
            </item>
            <item>
             <widget class="QPushButton" name="selByNm_bt">
              <property name="text">
               <string>根据名字选择</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="new_frm">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QPushButton" name="rnmT_bt">
               <property name="text">
                <string>Rename Tools</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="waiting_bt">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>。。。</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="old_frm">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <item row="2" column="1">
                <widget class="QPushButton" name="emgrp_tidy_bt">
                 <property name="text">
                  <string>自动清理空组</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="inac_tidy_bt">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="anlab_tidy_bt">
                 <property name="text">
                  <string>模型添加ct_an标记</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="nm_tidy_bt">
                 <property name="text">
                  <string>Object Name 整理</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="set_tidy_bt">
                 <property name="text">
                  <string>标记set组</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="suff_tidy_bt">
                 <property name="text">
                  <string>选取添加 _ 后缀</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="nmsp_tidy_bt">
                 <property name="text">
                  <string>name space 工具</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="dsly_tidy_bt">
                 <property name="text">
                  <string>display Layer 清理</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>文件检查</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="nm_chk_bt">
          <property name="text">
           <string>Check 命名</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="prxTr_chk_bt">
          <property name="text">
           <string>Check Proxy位移</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="smthSet_chk_bt">
          <property name="text">
           <string>Check smoothSet</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="flg_chk_bt">
          <property name="text">
           <string>Check 标记属性</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="ins_chk_bt">
          <property name="text">
           <string>Check Instance</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="tautonymyShp_chk_bt">
          <property name="text">
           <string>Check Shape重名</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="tautonymyMsh_chk_bt">
          <property name="text">
           <string>Check Mesh同名</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="sm_chk_bt">
          <property name="text">
           <string>Check Smooth</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="nmsp_chk_bt">
          <property name="text">
           <string>Check Namespace</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="tautonymy_chk_bt">
          <property name="text">
           <string>Check 重名</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="ref_chk_bt">
          <property name="text">
           <string>Check 参考</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="rndSta_chk_bt">
          <property name="text">
           <string>Check RenderState</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="fnm_chk_bt">
          <property name="text">
           <string>Check 面数</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="all_chk_bt">
          <property name="text">
           <string>...Check All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
